jss.spec

```diff
@@ -6,9 +6,9 @@ Summary:        Java Security Services (JSS)
 URL:            http://www.dogtagpki.org/wiki/JSS
 License:        MPLv1.1 or GPLv2+ or LGPLv2+
 
-Version:        4.7.2
-Release:        1%{?_timestamp}%{?_commit_id}%{?dist}
-#global         _phase -a1
+Version:        4.8.0
+Release:        0.1%{?_timestamp}%{?_commit_id}%{?dist}
+global         _phase -a1
```

Comment:
Sorry, missed this one. You need to change it to `%global`:
```
%global           _phase -alpha1
```
The `Release` number is used in RPM, the `_phase` is used in the source tarball. Perhaps they can be combined somehow.

Reply:
Yep, thanks @edewata!